src/modules/client/components/contracts/ContractEndingModal.vue

```diff
@@ -62,6 +66,13 @@ export default {
         this.validations.otherMisc.$reset();
       }
     },
+    async updateAndResetOtherMisc (event, prop) {
+      await this.update(event, prop);
```

Comment:
dans la logique, si tu mets un `await` ici il fait que tu définisses la méthode `update` avec un `async`
et pour mettre un `async` sur `update`, il faut mettre un `await` sur `$emit`
du coup peux tu vérifier que `$emit` est bien asynchone ?

Reply:
visiblement pas besoin de await devant un emit en effet, et ça a l'air de marcher correctement sans, non ?

Reply:
tu as pu trouver un peu de doc dessus ?

Reply:
pas de doc mais des stackoverflow et autre forums : 
https://forum.vuejs.org/t/do-something-after-emit-has-finished-successful/10663/7

https://stackoverflow.com/questions/54316518/is-vues-emit-async-operation/54316644